docs/gci-git-guide.md

```diff
@@ -0,0 +1,1041 @@
+# GCI Git & GitHub Guide
+
+This a shortened version of zulip's git guide specifically focused on GCI
+students. You can read the original version [here][zulip-git-guide].
```

Comment:
Is this supposed to be broken, so we can insert the appropriate link once it's in a good location?